### PR TITLE
Autoyast error new tags

### PR DESCRIPTION
**Pull request by asmorodskyi**

### CODE REVIEW

**Comment by asmorodskyi** [comment]
proof run http://kimball.arch.suse.de/tests/422

File: tests/autoyast/installation.pm [hunk 1]
```diff
@@ -55,32 +54,23 @@ sub save_logs_and_continue {
...
     send_key "ret";
     wait_idle(5);
 }
 
 sub run {
-    my @needles = ("bios-boot", "autoyast-error", "reboot-after-installation", "linuxrc-install-fail");
-    push @needles, "autoyast-confirm"        if get_var("AUTOYAST_CONFIRM");
-    push @needles, "autoyast-postpartscript" if get_var("USRSCR_DIALOG");
-    if (get_var("AUTOYAST_LICENSE")) {
+    my @needles = ('bios-boot', 'nonexisting-package', 'reboot-after-installation', 'linuxrc-install-fail');
```
Comment (okurz) [inline tests/autoyast/installation.pm]: can be further simplified to `my @needles = qw(bios-boot nonexisting-package reboot-after-installation linuxrc-install-fail)`

File: tests/autoyast/installation.pm [hunk 2]
```diff
@@ -22,7 +22,6 @@ use testapi;
 use utils;
 
 my $confirmed_licenses = 0;
-my $stage;
```
Comment (okurz) [inline tests/autoyast/installation.pm]: why do you delete the stage?

Comment (asmorodskyi) [inline tests/autoyast/installation.pm]: why we need it ? I don't see any functionality loss. 
before we had  :
installation-y2logs-stage1_error1.tar.bz2
installation-y2logs-stage1_error2.tar.bz2
installation-y2logs-stage2_error3.tar.bz2

and after my commit we will have :
installation-y2logs-error1.tar.bz2
installation-y2logs-error2.tar.bz2
installation-y2logs-error3.tar.bz2

anyway to understand what "stage1" / "stage2" suppose to mean you need to read the code. And if you read my code in a same you will be able to understand meaning of error1 / error2 / error3

**Comment by okurz** [comment]
LGTM but sorry, I don't trust the github GUI conflict resolution merge commit I created. Please rebase locally, update PR and merge.

**Pull request merged**